common/include/pcl/common/impl/centroid.hpp

```diff
@@ -43,6 +43,8 @@
 #include <pcl/common/centroid.h>
 #include <pcl/conversions.h>
 #include <pcl/common/point_tests.h> // for pcl::isFinite
+#include <pcl/common/transforms.h>
```

Comment:
I believe this is not necessary any more?

Reply:
yes I forgot, i'll delete it